influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -718,59 +735,75 @@ mod tests {
         wbuf.create_last_cache(db_name, tbl_name, Some(cache_name), None, None, None, None)
             .await
             .unwrap();
-        // Check that the catalog was persisted, without advancing time:
-        let object_store = wbuf.persister.object_store();
-        let catalog_json = fetch_catalog_as_json(
-            Arc::clone(&object_store),
-            wbuf.persister.host_identifier_prefix(),
+
+        // load a new write buffer to ensure its durable
+        let wbuf = WriteBufferImpl::new(
+            Arc::clone(&wbuf.persister),
+            Arc::clone(&wbuf.time_provider),
+            Arc::clone(&wbuf.buffer.executor),
+            WalConfig {
+                level_0_duration: Level0Duration::new_1m(),
+                max_write_buffer_size: 100,
+                flush_interval: Duration::from_millis(10),
+                snapshot_size: 1,
+            },
```

Comment:
Cool, so this just replays the WAL which has the op to create the cache in it, and therefore the resulting catalog is ✅